scripts/build/windows/install-build-pre-requisites.ps1

```diff
@@ -133,6 +133,24 @@ function Install-Docker() {
    Write-Host("installing docker for desktop completed")
 }
 
+function Install-cmetrics() {
+    #Install flex and bison
+    choco install -y winflexbison3
+
+    $destinationPath = Join-Path -Path $env:SYSTEMDRIVE -ChildPath "cmetrics"
+    New-Item -Path $destinationPath -ItemType "directory" -Force -ErrorAction Stop
+    #Clone cmetrics repo and install cmetrics and all its dependencies
+    git clone --recursive https://github.com/calyptia/cmetrics.git
+	cd cmetrics
+	git checkout v0.6.0
```

Comment:
in main.ps1, we are using  v0.1.7 where as here we are using 0.6.0. what's the reason for using different versions?

Reply:
Both are different repositories....This is the downstream cmetrics library and the other one is the go library based on cmetrics...